elearning-backend/app/Http/Controllers/API/AdminWordsController.php

```diff
@@ -9,17 +9,55 @@
 
 class AdminWordsController extends Controller
 {
+    // Parse choices
+    private function getChoices($choices)
+    {
+        return json_decode($choices);
+    }
```

Comment:
what is this for? this is only decoding and do nothing else. not useful function